code/game/objects/items/scanners.dm

```diff
@@ -18,9 +18,6 @@ REAGENT SCANNER
 	w_class = WEIGHT_CLASS_SMALL
 	item_state = "electronic"
 
-	matter = list("metal" = 150)
```

Comment:
Is removing the custom values of these items intentional?

Reply:
Yes, I made some items more generic, being inherited from the base /item path. Can tweak if you'd like, but these weren't used much if at all, this was done to make future balancing simpler.

Reply:
No need if it's intentional. This is fine.